docs/api.md

```
@@ -898,6 +905,37 @@ GET /api/interviews
 
 ---
 
+### 면접 삭제
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)